app/Http/Requests/VoteRequest.php

```diff
@@ -63,8 +63,7 @@ public function rules()
     public function messages()
     {
         return [
-            'phone.phone' => 'That doesn\'t look like a real phone number!',
-            'birthdate.date' => 'Enter your birthday MM/DD/YYYY!',
```

Comment:
Yes, should've probably broken that into it's own commit though. Date validation was changed in #474 to use the custom `localized_date` validator which returns the proper localized syntax so this was just some leftover cruft.